server/src/main/java/org/opensearch/index/search/stats/SearchStats.java

```diff
@@ -163,6 +163,8 @@ public static class Stats implements Writeable, ToXContentFragment {
         private long pitTimeInMillis;
         private long pitCurrent;
 
+        private long searchIdleWakenUpCount;
```

Comment:
Maybe `searchIdleRetryCount` or `searchIdleReactivateCount` would be more readable and still represent the same idea.